.github/workflows/docs-preview.yml

```diff
@@ -0,0 +1,20 @@
+---
+name: docs-preview
+
+on:
+  pull_request_target:
+    types: [opened]
+
+permissions:
+  pull-requests: write
+
+jobs:
+  doc-preview-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: elastic/docs/.github/actions/docs-preview@main
```

Comment:
I think I changed `master` to `main` since that's what we use in the ingest-docs repo, but I understand now that this applies to the docs repo instead.